tests/lib/state.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+SNAPD_STATE_PATH="$SPREAD_PATH/snapd-state"
+
+get_boot_path() {
+    BOOT=""
```

Comment:
Not sure if it's worth it but it may be better if `get_boot_path` did not set a global variable, but echoed something instead. The code would then look like this:
```
save_all_snap_state() {
    boot_path="$(get_boot_path)"
    test -n "$boot_path" || exit 1
    ...
}
```

Reply:
done